include/flatbuffers/util.h

```diff
@@ -633,6 +633,12 @@ inline bool EscapeString(const char *s, size_t length, std::string *_text,
   return true;
 }
 
+inline std::string FirstToLower(std::string in) {
```

Comment:
Why is this in util.h? This is also an inefficient way of doing it, better keep this code to where it is actually used.

Reply:
Ok you are right. I do that in the idl_gen_general.cpp